packages/devtools_app/lib/src/theme.dart

```diff
@@ -226,36 +223,53 @@ extension DevToolsColorScheme on ColorScheme {
       isLight ? Colors.grey[300] : Colors.grey[700];
 
   // Title of the hover card.
-  TextStyle get hoverTitleTextStyle => const TextStyle(
-        color: Colors.black,
-        fontWeight: FontWeight.bold,
-        fontSize: 16,
+  TextStyle get hoverTitleTextStyle => TextStyle(
+        color: isLight ? Colors.black87 : Colors.white54,
+        fontWeight: FontWeight.normal,
```

Comment:
is FontWeight.normal the default? is it necessary to specify this parameter everywhere?

Reply:
No, looks like bold to me.